docs/source/cn/guides/overview.md

```diff
@@ -0,0 +1,130 @@
+<!--⚠️ 请注意，此文件为 Markdown 格式，但包含我们文档生成器的特定语法（类似于 MDX），可能无法在您的 Markdown 查看器中正确渲染。
+-->
+
+# 操作指南
+
+在本节中，您将找到帮助您实现特定目标的实用指南。
+查看这些指南，了解如何使用 huggingface_hub 解决实际问题：
+
+<div class="mt-10">
+  <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
+
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg"
+       href="./repository">
+      <div class="w-full text-center bg-gradient-to-br from-indigo-400 to-indigo-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">
+        仓库
+      </div><p class="text-gray-700">
+        如何在 Hub 上创建仓库？如何配置它？如何与之交互？
+      </p>
+    </a>
+
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg"
+       href="./download">
+      <div class="w-full text-center bg-gradient-to-br from-indigo-400 to-indigo-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">
+        下载文件
+      </div><p class="text-gray-700">
+        如何从 Hub 下载文件？如何下载仓库？
+      </p>
+    </a>
+
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg"
+       href="./upload">
+      <div class="w-full text-center bg-gradient-to-br from-indigo-400 to-indigo-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">
+        上传文件
+      </div><p class="text-gray-700">
+        如何上传文件或文件夹？如何对 Hub 上的现有仓库进行更改？
+      </p>
+    </a>
+
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg"
+       href="./search">
+      <div class="w-full text-center bg-gradient-to-br from-indigo-400 to-indigo-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">
+        搜索
+      </div><p class="text-gray-700">
+        如何高效地搜索超过 200k+ 个公共模型、数据集和空间？
```

Comment:
I feel that it would be better to use the English word "Space" here, as it is a specific name. Keeping the original term would be more appropriate.